src/gui/vector/qgsstackeddiagramproperties.cpp

```diff
@@ -189,7 +189,7 @@ void QgsStackedDiagramProperties::syncToLayer()
       const auto renderers = stackedDiagramRenderer->renderers();
       for ( const auto &renderer : renderers )
```

Comment:
Can you avoid the use of auto for things like this? It goes against current QGIS coding guidelines, which is only to use auto for complex types like iterators.

Reply:
Sure! Thanks for the suggestion.

Reply:
Addressed in https://github.com/qgis/QGIS/pull/59123/commits/3bbad5495b61cbc8bc297c01f6c8cce831ef8bd8